packages/lib/src/spatial-navigation/components/virtualizedGrid/SpatialNavigationVirtualizedGrid.tsx

```diff
@@ -27,10 +27,8 @@ type SpatialNavigationVirtualizedGridProps<T> = Pick<
     itemHeight: number;
     header?: JSX.Element;
     headerSize?: number;
-    /** How many rows are RENDERED (virtualization size) */
-    numberOfRenderedRows: number;
-    /** How many rows are visible on screen (helps with knowing how to slice our data and to stop the scroll at the end of the list) */
-    numberOfRowsVisibleOnScreen: number;
+    /** How many rows are RENDERED ADDITIONALLY to those already visible (impacts virtualization size) */
```

Comment:
you can add details in this TSDoc 😁  (default value, what it is for jump-on-scroll vs stick-to-*)